doc/source/features/online-schema-changes.rst

```diff
@@ -0,0 +1,82 @@
+.. _online-schema-changes:
+
+===================================================
+Online Schema Changes
+===================================================
+
+A failure in a schema change can cause your cluster to drop nodes and lose data. |Percona XtraDB Cluster| supports the following methods for making online schema changes:
+
+.. list-table::
+    :widths: 20 60
+    :header-rows: 1
+
+    * - Method 
+      - Description
+    * - `TOI`
+      - Total Order Isolation
+        
+        The default DDL replication method for the :ref:`wsrep osu method` variable. The isolation of the DDL statement guarantees consistency.
+        
+        The node detects a DDL statement in the Parser stage and sends the statement to all nodes. All nodes wait for any earlier transactions to commit, and then execute the statement.
+        
+        All other writes must wait until the DDL statement is executed. While a DDL statement is running, any long-running transactions in progress and using the same resource receive a deadlock error at commit and are rolled back. There is no way to verify if individual nodes have succeeded in processing the DDL statement. 
+    * - `RSU`
+      - Rolling  Schema Upgrade
+
+        A DDL statement are processed on each nod locally. This action guarantees high availability. The node desynchronizes with the cluster and disables flow control while executing the DDL statement. The rest of the cluster is not affected. After the statement execution, the node applies delayed events and synchronizes with the cluster. 
+
+        The cluster continues to operate, although during the process, some nodes may be updated and some nodes may have the older schema structure.
+
+        For this method, the ``gcache`` must be large enough to store the data for the duration of the DDL change. 
+    * - `NBO`
+      - Non-Blocking Operation
```

Comment:
@georgelorchpercona shouldn' t NBO be an experimental feature in the next release?

Reply:
@dutow I thought we discussed that this is not practical to call "Tech Preview" as it is pretty core changes that can not be avoided regardless of enabled or not.

Reply:
OK, yes, this feature is going to be tagged as "Tech Preview" in 8.0.25, if no serious issues present themselves, it can go to GA in 8.0.27

Reply:
Added.